src/importation/index.js

```diff
@@ -0,0 +1,110 @@
+// Importation model.
+// "Importation" because "import" is a reserved keyword.
+
+import _ from "lodash";
+import PropTypes from "prop-types";
+import Report from "./report";
+import Status from "./status";
+import vcard from "./vcard";
+
+const EMPTY_FILE = "importation.empty_file";
+const INVALID_FILE_TYPE = "importation.invalid_file_type";
+
+const propType = PropTypes.shape({
+  file: PropTypes.instanceOf(File),
+  fileIssue: PropTypes.oneOf([EMPTY_FILE, INVALID_FILE_TYPE]),
+  report: Report.propType,
+  status: Status.propType.isRequired
+});
+
+const VALID_FILE_TYPES = vcard.FILE_TYPES;
+
+export default {
+  EMPTY_FILE,
+  INVALID_FILE_TYPE,
+  VALID_FILE_TYPES,
+  canRetry,
+  canRun,
+  filename,
+  init,
+  propType,
+  run,
+  selectFile,
+  unselectFile
+};
+
+function init() {
+  return {
+    status: Status.UNCONFIGURED,
+    file: undefined,
+    fileIssue: undefined,
+    report: undefined
+  };
+}
```

Comment:
I was afraid that, in case some buggy code mutates the initial object, anything else using it as a starting point would break in unexpected ways. And the cost of calling a function seemed negligible since we shouldn't be massively creating new importations. Other *values* (e.g. const sets / lists) didn't bother me so much because they were not supposed to be modified, even through cloning. But maybe I'm just being paranoid because JavaScript doesn't use immutable datastructures by default.